core/api.txt

```diff
@@ -427,6 +427,9 @@ ion-datetime,css-prop,--background-rgb
 ion-datetime,css-prop,--title-color
 ion-datetime,css-prop,--wheel-fade-background-rgb
 ion-datetime,css-prop,--wheel-highlight-background
+ion-datetime,part,calendar-day
+ion-datetime,part,calendar-day active
```

Comment:
Threading: Can we add a test that checks the focus state (both default and customized w/ a shadow part?)

Reply:
Added tests here: https://github.com/ionic-team/ionic-framework/pull/27641/commits/198bd3d4ead9a4c41bbbe6fcbf36e62985c436d0
Screenshots were added in: https://github.com/ionic-team/ionic-framework/pull/27641/commits/aa3881ec90c546fcc547915eb9074edbb8b1283e

Realized we don't need RTL and removed (:joy:) in: https://github.com/ionic-team/ionic-framework/pull/27641/commits/cbba93741000f3275e83975ff8be521e6020c6ad

LMK if you want me to change anything!